decidim-admin/app/packs/stylesheets/decidim/admin/_item_show.scss

```diff
@@ -12,7 +12,7 @@
   &-title {
     @apply text-xl font-bold block sm:flex items-center justify-between gap-4 w-full first-letter:capitalize first:[&>div]:ml-0 first:[&>*]:ml-auto;
 
-    .dropdown-pane {
+    .dropdown-pane li a {
```

Comment:
We need to remove this as, currently, this is not making the font-normal for the elements inside the `.dropdown-pane`. 

This is how it looks with your changes:

![Screenshot of the dropdown pane](https://github.com/decidim/decidim/assets/717367/18469dba-b0a9-46dc-967b-a6fbf67fddba)

Without this change, it looks as it should:

```suggestion
    .dropdown-pane {
```